### DISCUSSION TITLE
Code to connect to a particular server?

### DISCUSSION THREAD

**Discussion by Phamine** [bubuntux/nordvpn#169] [Q&A] [answered]
I'd like to connect to this server:

us6824.nordvpn.com

What would be the correct syntax for below?

    environment:                              
      - CONNECT=??

**Comment by bubuntux** [comment, marked as answer]
check the documentation,
Provide a [server] argument to connect to a specific server. For example: jp35 , Full List

so it should be CONNECT=us6824
